src/packet/BinaryWriter.js

```diff
@@ -8,7 +8,7 @@
  */
 
 function BinaryWriter() {
-    this._buffer = new Buffer(1e5);
+    this._buffer = Buffer.allocUnsafe(1e5);

```

Comment:
[From the nodejs v7.4.0 docs:](https://nodejs.org/api/buffer.html#buffer_buffer_from_buffer_alloc_and_buffer_allocunsafe)
> In versions of Node.js prior to v6, `Buffer` instances were created using the `Buffer` constructor function, which allocates the returned `Buffer` differently based on what arguments are provided:
> ...

This will generate a ReferenceError if node version is below v6.
Even though the current LTS v6.9.4, add a failsafe just in case:
```js
    this.buffer = parseInt(process.version[1]) < 6 ? new Buffer(1e5) : Buffer.allocUnsafe(1e5);
```